tilbakekreving/application/src/main/kotlin/tilbakekreving/application/service/consumer/KnyttKravgrunnlagTilSakOgUtbetalingKonsument.kt

```diff
@@ -56,14 +56,16 @@ class KnyttKravgrunnlagTilSakOgUtbetalingKonsument(
             )
             nonEmptyListOf(it)
         }.flatMap {
-            it.flattenOrAccumulate().map { }
+            it.flattenOrAccumulate().map {
+                it.mapNotNull { it.second }
```

Comment:
Hva er vitsen med å returnere dette når it.second alltid settes til null på linje 98?